JuliaInterface/read.g

```diff
@@ -11,6 +11,8 @@ dirs:= DirectoriesPackageLibrary( "JuliaInterface", "julia" );
 
 JuliaIncludeFile( Filename( dirs, "gaptypes.jl" ) );
 
+_InitializeGAPFFEType();
```

Comment:
Why not do that in `InitKernel`?

Reply:
Because we need to have the GAP module first, which gets loaded into Julia after the package `.so` is loaded into GAP.

Reply:
Fair enough. It would be helpful if there was a comment somewhere explaining that, though...